src/main/resources/se/diabol/jenkins/pipeline/portlet/DeliveryPipelineViewPortlet/portlet.jelly

```diff
@@ -0,0 +1,19 @@
+
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:dp="/hudson/plugins/view/dashboard" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+    <dp:decorate portlet="${it}">
+        <tr><td>
+            <div>
+                <st:include page="main.jelly" it="${it.deliveryPipelineView}" />
+                <style type="text/css">
+                    div.pipeline-main {
+                        border:none !important;
+                    }
+                    div.pagination {
+                        height: 1px !important;
+                        line-height: 1px !important;
+                    }
+                </style>
+            </div>
+        </td></tr>
+    </dp:decorate>
+</j:jelly>
```

Comment:
Dashboard View is a new view implementation that provides a dashboard / portal-like view for Jenkins.
We can create a view and add portlets, several plugins which records data from Jenkins gives some kind of reports.
Below is some kind of dashboard and how this DPP portlet will be useful in Dashboards page.
![dashboard](https://cloud.githubusercontent.com/assets/8586903/16709367/976bf96c-462c-11e6-800f-76688a5bafe9.png)